mongodbatlas/organizations.go

```diff
@@ -40,6 +40,12 @@ type OrganizationsServiceOp service
 
 var _ OrganizationsService = &OrganizationsServiceOp{}
 
+// OrganizationsListOptions filtering options for organizations
+type OrganizationsListOptions struct {
+	Name string `url:"name,omitempty"`
```

Comment:
Since you mention Ops Manager, Ops manager also has a boolean, `includeDeletedOrgs`, I've been wanting to fix this Listing option to add that one in particular but since we are breaking this I wouldn't mind taking the extra step here.
If you decide to add it be aware that it defaults to true so `omitempty` may get tricky

Reply:
I think using a bool pointer can help get around the `omitempty` business. According to the docs, if the `includeDeletedOrgs` param is missing in the url, then it'll still default to `true`.

<img width="806" alt="image" src="https://user-images.githubusercontent.com/17665837/94949537-13a06a00-04af-11eb-823d-684c5bc51352.png">


I added in that change and made a test to show that `setListOptions` will properly omit an empty bool pointer. Please let me know if you need anything else for this PR.

Reply:
thanks for adding this, yes, making it a pointer should work